tuxedo-core/src/utxo_set.rs

```diff
@@ -1,25 +1,23 @@
-//! These three functions comprize what I sketched out in the UtxoSet trait in element the other day
-//! For now, this is complicated enough, so I'll just leave them here. In the future it may be wise to
-//! abstract away the utxo set. Especially if we start doing zk stuff and need the nullifiers.
+//! TODO explanation this type
```

Comment:
:eyes:

Reply:
Ok ill leave it in wasnt sure what we should do here.